tests/model.test.ts

```diff
@@ -130,6 +130,19 @@ describe("Model", () => {
     expect(receivedCommands).not.toContain("COPY");
   });
 
+  test("Core plugins handle don't receive UI commands", () => {
+    const receivedCommands: CommandTypes[] = [];
+    class MyCorePlugin extends CorePlugin {
+      handle(cmd: CoreCommand) {
+        receivedCommands.push(cmd.type);
+      }
+    }
+    corePluginRegistry.add("myCorePlugin", MyCorePlugin);
```

Comment:
you should clean the plugin registry because it leads to a memory leak. Every model instantiated after this test will have this plugin and `receivedCommands` will grow and grow with every dispatched command
(I'm sure there are other places where we don't properly clean test plugins)